src/Annotation/AsymmetricKeysGenerator.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\pubkey_encrypt\Annotation\AsymmetricKeysGenerator.
+ */
+
+namespace Drupal\pubkey_encrypt\Annotation;
+
+use Drupal\Component\Annotation\Plugin;
+
+/**
+ * Defines a Asymmetric Keys Generator annotation object.
+ *
+ * @see \Drupal\pubkey_encrypt\Plugin\AsymmetricKeysManager
+ * @see plugin_api
+ *
+ * @Annotation
+ */
+class AsymmetricKeysGenerator extends Plugin {
```

Comment:
I modeled all these classes upon the ones provided in Encrypt module. The classes there don't start with the module name so that's why I didn't do it. What do you say?

Also I missed adding  `@ingroup pubkey_encrypt` into the Annotation comment. Maybe that's required?